aiohttp/web_jobs.py

```diff
@@ -0,0 +1,223 @@
+import asyncio
+import sys
+import traceback
+from collections import deque
+from collections.abc import Container
+
+import async_timeout
+
+from .helpers import ensure_future
+
+
+__all__ = ['JobRunner']
+
+
+class Job:
```

Comment:
I know Job name is sweet and short, but it introduces a new entity on the board. How about to stay with Task name? Internally, you still uses `_task`, not something else.